main.go

```diff
@@ -29,30 +29,36 @@ func main() {
 
 	rootName := getModulePackageName()
 
+	// write report header
 	var buf bytes.Buffer
```

Comment:
One might argue this should be https://pkg.go.dev/strings#Builder given how it is used. :D

Reply:
Oh geez - well spotted - I'll add this in now 👍

Reply:
Fixed in 0ef64bb 👍